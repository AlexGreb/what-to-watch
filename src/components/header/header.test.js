import React from 'react';
import renderer from 'react-test-renderer';
import Header from './header.jsx';

describe(`Header`, () => {
  it(`Should render correctly header`, () => {
    const tree = renderer.create(<Header/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

