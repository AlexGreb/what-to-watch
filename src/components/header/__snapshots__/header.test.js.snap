// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render correctly header 1`] = `
<DocumentFragment>
  <header
    class="page-header movie-card__head"
  >
    <div
      class="logo"
    >
      <a
        class="logo__link"
      >
        <span
          class="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          class="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          class="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      class="user-block"
    >
      <div
        class="user-block"
      >
        <a
          class="user-block__link"
          href="sign-in.html"
        >
          Sign in
        </a>
      </div>
    </div>
  </header>
</DocumentFragment>
`;
